ftp/src/test/java/akka/stream/alpakka/ftp/CommonStageTest.java

```diff
@@ -17,14 +17,15 @@
 import akka.util.ByteString;
 import org.junit.Assert;
 
+import java.time.Instant;
 import java.util.concurrent.CompletionStage;
 import java.util.concurrent.TimeUnit;
 import java.util.function.Function;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
 
-interface CommonFtpStageTest extends FtpSupport, AkkaSupport {
+interface CommonStageTest extends BaseSupport, AkkaSupport {
```

Comment:
Any particular reason for the rename? (not renaming would keep a smaller diff)

Reply:
I initially removed some 'Ftp' from trait names that were common to FTP and SFTP, mostly for me to understand better the overall file organization. I agree that it is not especially needed here, I'll revert.

Reply:
Wait, did you also mean revert BaseSupport -> FtpSupport?

Reply:
No. :-)